src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -97,10 +97,18 @@ private void ExecuteSearchElement(ListBoxItem listBoxItem)
                 if (searchElement.CreateAndConnectCommand.CanExecute(port.PortModel))
                 {
                     searchElement.CreateAndConnectCommand.Execute(port.PortModel);
+                    var selectedNodeName = (searchElement.Model is Search.SearchElements.NodeModelSearchElement) ? string.Format("{0}, {1}", searchElement.FullName, searchElement.Assembly) : searchElement.Model.CreationName;
+                    var originalNodeName = port.NodeViewModel.NodeModel.CreationName;
```

Comment:
Sorry I wasn't clear in the last comment. originalNodeName will should be also be checked for NodeModelSearchElement type or ZeroTouchSearchElement.

Reply:
Also just realized that we also have a third type CustomNodeSearchElement. ML API won't be returning any CustomNodeSearchElement but I think the node type match will check for custom nodes as well. So better to check for both types explicitly.

Reply:
You can differentiate based on the NodeModel inside NodeViewModel, it will be either DSFunctionBase or NodeModel for those 2 cases

Reply:
Gotcha, do you know what should we return in both cases for Custom nodes?

Reply:
For custom nodes, we are storing it in the same format as the NodeModel nodes.

Reply:
Ok, then I should check for ZeroTouch node first